\mainmatter
\part{Introduction}
\fancyhead[RE]{CHAPTER \thechapter}

Structural Biology
==================

This thesis is about 'assignment methods for large proteins in solid state nuclear magnetic resonance spectroscopy ...'. Before bombarding you with details I would like to give a short introduction into the context of this research.

Nuclear magnetic resonance, or NMR, spectroscopy is a very standard method for the analysis of chemical substances. The general idea behind the technique is, like in any other form of spectroscopy, to measure a difference in energy between different states. In the case of NMR spectroscopy the different states are generated by bringing the sample in a large magnetic field. All atomic nuclei have a quantum mechanical property called 'spin'. In a magnetic field the spin of some atomic nuclei, for example the nucleus of the normal hydrogen 1H isotope, will be either pointed along the magnetic field or against it. The energy of the spins that are aligned with the magnetic field is slightly lower than that of those that are pointed against it. This is the energy difference that is exploited by NMR spectroscopy and leads to a signal in a NMR spectrum. The energy difference is directly related to the strength of the magnetic field. However the magnetic field that is perceived by a given spin is not only only determined by the external field of the magnet. Also the direct chemical environment around the spin influences the local magnetic field. Therefor every nucleus in a molecule that has a slightly different chemical environment gives rise to a peak on a different position in the spectrum. By careful analysis of these spectra, often combined with mass spectrometry, it has been possible for decades to determine the composition and structure of small organic molecules.
The use of NMR spectrometry in structural biology, which this thesis is about, is a little bit different. We generally already know the chemical composition of the molecule of interest. In case of a protein we know its amino acid sequence. In case of RNA or DNA we know the nucleotide sequence. Of course we also know the chemical composition of the amino acids and nucleotides respectively. The only missing information is the shape it adopts in three dimensional space and how this shape changes over time. Structural biology is essentially about reverse-engineering on a bio-molecular level. The central dogma in the field is that the structures of bio-molecules such as DNA, RNA and proteins can tell something about their function and how they fulfill this function. Of course this is interesting from a purely scientific point of view. However, understanding the biological mechanisms that are connected to diseases can help developing therapies.

The energy difference used in NMR is only very small compared to the thermal energy at biologically relevant temperatures. Therefor NMR is an inherently insensitive method. A measurable signal can only be generated by the measuring millions of molecules at the same time. As will be explained later, in one way or another all difficulties with this method lead back to this fact.

In both liquid and solid samples, every molecule in the sample has a different spacial orientation in respect to direction of the static field of the magnet.
The first structure of a protein,... was solved in 19.. by x-ray crystallography.


> *This* is a quotation block: Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.




Membrane Proteins
=================

The lipids in a biological membrane are very diverse and differ between organisms and cell types. The membrane is very crowded with membrane proteins, often the amount of protein (in terms of weight) is larger than the amount of lipids. In E. coli membranes 75 weight percent can be attributed to proteins (Lehninger). The lipids in the two leaflets that compose the lipid bilayer are often distinct. This is also the case in the outer membrane of E. coli. The outer leaflet is composed of lipopolysaccharides (LPS), and the inner leaflet of the more usual phospholipids. This influences the membrane thickness, which is with 24 Ã… a little thinner than would be expected....


In general membrane proteins can be divided in several classes, 7 transmembrane, beta-barrel....

The topology of membrane proteins can often be predicted based on the residue sequence. For instance, 20 or more hydrofobic residues is a row is quite likely to form a transmembrane helix. Also other types of residues give a hint to the topology of a membrane protein. For instance tryptophan residues are often found at the interface between the transmembrane and the non-transmembrane part of the protein since they have a large flat aromatic ring system....



![Distribution of sequence lengths of all proteins in the uniProt database tagged as membrane protein. The amount of proteins with sequence lengths similar to OmpG, 281 residues, is very large, making OmpG a relevant model system. \label{sequence_lengths}](figures/membrane_protein_sequence_lenth_distribution.svg)





Sequential Assignment of solid state NMR spectra
================================================

In most NMR studies very little can be done before the chemical shifts of the nuclei that are interesting in the context of the biological question are known. Sometimes those are only a few. However if the goal is to calculate the structure and study the overall dynamics of the protein a fairly complete mapping between resonance frequencies and nuclei in the molecule has to be present. This mapping process is referred to as sequential assignment and often is the most time consuming part of a NMR study. The general idea behind sequential assignment methodologies is always the same: the graph that arises from a set of correlation spectra is mapped on the molecular topology. In most (but not all) methodologies to find this mapping, first parts of the total signal pattern are identified that correspond to individual residues. By finding connectivities between these signal patterns a larger signal pattern that belongs to a set of sequentially connected residues is created. Because the nuclei in different amino acids give rise to a different combination of chemical shifts, the sets of signals can be classified down to a few or sometimes even one type of amino acid. Then this pattern is mapped to a subsequence in the protein that matches these possible residue type assignments.

From now on I will call the collection of resonance frequencies that belong to one residue a spin system. The term **spin system** is often used in NMR in a somewhat less confined sense, meaning a set of resonances that are in some way influenced by one another. However, since this thesis will deal with sequential assignment for a large extent, it is good to have quick way to describe this object. Also the CCPNMR Analysis software, that is used to analyze NMR spectra, uses the term as I just defined it.

In order to get a unique match between the potential residue types of a sequential stretch of spin systems and a subsequence in the protein, the stretches should in general be long enough. I.e. the longer a connected stretch is, the higher the chance there is only one possible location along the protein sequence were this stretch fits. Of course it highly depends on the length of the protein sequence how long it takes before this is the case. In figure {@fig:amount_of_unique_subsequences} the fraction of subsequences of lenght 1, 2 and 3 are plotted vs. the lenght of the protein. As can be seen though, even for very large proteins, connected stretches of 3 spin systems can in theory be uniquely matches to a subsequence in the majority of cases. This is of course under the assumption that each spin system can be unique typed to one amino acid type, which is not the case in practice since some amino acids give rise to very similar signal sets.



![Percentage of subsequences that is only present in the sequence once. Purple, orange and green correspond to subsequences of length 1,2 and 3 respectively. This plot is made using 1000 membrane protein sequences from the uniProt database. Every point represents one protein. As expected, the amount of single amino acids that only appear in the sequence once very quickly drops of with increasing sequence length. At the other side, even for the largest proteins still more than half of all triplets (subsequences of length 3) is completely unique in the sequence. Of course, the fact that the subsequence is unique does not necessarily mean that the subsequence can be distinguished from all other subsequences based on the chemical shifts of these residues. That also heavily depends on how many 13C chemical shifts of the spin systems are known, which is directly dependent on the type of spectra recorded.  \label{unique_subsequences}](figures/amount_of_unique_subsequences.svg){#fig:amount_of_unique_subsequences}



The real bottleneck in assigning larger proteins is chemical shift degeneracy. The larger the amount of NMR active nuclei in a protein, the smaller the average difference between the resonance frequences will be. This causes two problems that are very closely related, but I think should be considered both distinctly:

1. spectral crowding, the overlap of signals in spectra.
2. degeneracy of assignment of peaks

The first problem is very simple, spectra get very hard to interpret if signals are piled on top of each other. The second problem maybe needs a little more explanation. In order to connect spin systems to form a sequential stretch, cross peaks between them in correlation spectra are needed. However, if a crosspeak can be explained by multiple different resonance frequencies from multiple spin systems on each dimension, it gets a lot harder to build these stretches. It is not so much that this type of peak is necessarily really overlapping with another peak, it is the apparent overlap of peaks.

Of course in the end these problems barrel down to linewidth. If linewidths were infinitely narrow, none of these problems would be there. There would be no signal overlap and every cross-peak could be explained by the correlation of spins of exactly the same number as its dimensions. If this were the case, there would be a very clear one to one mapping between the signal set and the molecular topology.

To combat the problem of spectral crowding and assignment ambiguity of crosspeaks, three approaches are taken in this study:

1. isotopically labelling only a subset of the nuclei in the protein
2. spread the signal set over more dimensions
3. decrease linewidths

Over the last few years, solid state NMR has made an enormous amount of progress and also the methods we used in this project changed during this process. The NMR methods used have a large influences on the way sequential assignments are made. I start of with a section explaining the efforts we did to assign OmpG using ^13^C-detected NMR, because that is how this was generally done back in the old days, 5 years ago, and how the project started off. Afterwards I will go on showing the progress we made in this project using ^1^H-detected experiments. Although it might seem that the use of the latter method is more important, the results from the ^13^C-detected experiments are not just included for chronological completeness. As will be clear, a lot of information obtained from the ^13^C-detected spectra is still very valuable and there is until now not a method in ^1^H-detected solid state NMR that can fully obtain the situation. I have always used the spectra from both methods at the same time and in a complementary fashion. Therefor in the last section of this chapter I will discuss the combination of both types of spectra.




![Electron micrograph of the OmpG 2D crystals. In this specific case it is OmpG and E. Coli lipids in a 2:1 mass ratio in MES buffer at pH 6.3. The lipid-OmpG mix seems to make tubular shapes.](figures/OmpG_MES_pH63_70procentH_30procentD_30042014_B.png)








